assets/style.css

```diff
@@ -0,0 +1,500 @@
+:root {
+  --indigo-dye: #013a63ff;
+  --light-blue: #a9d6e5ff;
+  --powder-blue: #caf0f8ff;
+}
+
```

Comment:
Not a bad thing at all! I love using root variables so I love seeing this. Great work!